workspaces/hurl-client/hurl-client-extension/README.md

```
@@ -31,3 +31,51 @@ Content-Type: application/json
+GET https://api.example.com/profile
+Authorization: Bearer {{token}}
+HTTP 200
+```
+
+Select both cells above and run them together — `{{token}}` only resolves because the cell that captures it ran in the same invocation.
+
+## Variables
+
+To set your own variables (like a base URL or API key), create a file named `hurl.vars` in the same folder as your `.hurl` files:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)